### PR TITLE
Clarify how to run tests in container

### PR DESCRIPTION
**Pull request by epapbak**
# Description

Update README to clarify how to run the tests in docker container locally.

Fixes [CCXDEV-11468](https://issues.redhat.com/browse/CCXDEV-11468)

## Type of change

- Documentation update

## Testing steps

N/A

## Checklist
* [ ] Pylint passes for Python sources
* [ ] sources has been pre-processed by Black
* [ ] updated documentation wherever necessary
* [ ] new tests can be executed both locally and within docker container

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,7 +31,16 @@ The `POSTGRES_DB_NAME` environment variable is mandatory, as the different servi
 
 If you don't want to spin up the containers, you'll need to locally run the required services (database, Kafka, etc.). You may need to add or remove the `managed` tag in the `${SERVICE}_tests.sh`.
 
-> **_NOTE:_**  Don't forget to set up `PATH` environment variable correctly when tests are run outside containers. That environment variable needs to also contain directory with tested executable files (for example `insights-results-aggregator-cleaner` etc.)
+> **_NOTE:_**  Don't forget to set up `PATH` environment variable correctly when runnig tests both within and outside containers. That environment variable needs to contain the path with tested executable files (for example `insights-results-aggregator-cleaner` etc.).
+
+When running the tests within the docker containers, it is necessary to have said executable under the `$VIRTUAL_ENV_BIN` folder with the right permissions. For example, after compiling `insights-results-aggregator`:
```
Comment (juandspy) [inline README.md:36]: > it is necessary to have said executable

"said"? I don; t get it

Comment (juandspy) [reply README.md:36]: is it "safe" (?)

**Review by juandspy — APPROVED** [review]

**Review by JiriPapousek — APPROVED** [review]
That looks great - thanks a lot! :)

File: README.md (hunk 1 above)
Comment (epapbak) [reply README.md:36]: nono, it is "said". "Said executable", as in the previously mentioned

**Pull request merged**